src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4834,22 +4936,47 @@ export class StudyViewPageStore
                     );
                     //only invoke if there are filtered samples
                     if (chartInfo && this.hasFilteredSamples) {
-                        const result = await internalClient.fetchGenomicDataCountsUsingPOST(
-                            {
-                                genomicDataCountFilter: {
-                                    genomicDataFilters: [
-                                        {
-                                            hugoGeneSymbol:
-                                                chartInfo.hugoGeneSymbol,
-                                            profileType: chartInfo.profileType,
-                                        },
-                                    ] as any,
-                                    studyViewFilter: this.filters,
+                        let result = [];
+                        let getDisplayedValue;
+                        let params = {
+                            genomicDataCountFilter: {
+                                genomicDataFilters: [
+                                    {
+                                        hugoGeneSymbol:
+                                            chartInfo.hugoGeneSymbol,
+                                        profileType: chartInfo.profileType,
+                                    },
+                                ] as any,
+                                studyViewFilter: this.filters,
+                            },
+                        } as any;
+
+                        if (
+                            chartInfo.profileType ===
+                            MolecularAlterationType_filenameSuffix.MUTATION_EXTENDED
+                        ) {
+                            params = {
+                                ...params,
+                                $queryParameters: {
+                                    projection:
+                                        chartInfo.mutationOptionType ===
```

Comment:
what's reason for this? put comment explaining why one case needs DETAILED

Reply:
There's no DETAILED condition in here, code updated
```
params = {
    ...params,
    $queryParameters: {
        projection: 'SUMMARY'
    },
};
```